arraycontext/impl/pytato/__init__.py

```diff
@@ -136,13 +139,30 @@ def _get_fake_numpy_namespace(self):
         from arraycontext.impl.pytato.fake_numpy import PytatoFakeNumpyNamespace
         return PytatoFakeNumpyNamespace(self)
 
+    @abc.abstractproperty
+    def frozen_array_types(self) -> Tuple[Type, ...]:
```

Comment:
Underscore this, since it's pytato-derived-only?

Reply:
Sure thing https://github.com/inducer/arraycontext/compare/f4ab671ce9eb96e9dd2397d9a77d5e7ac7c9be36..971df793aed6b5b107be09588c0791a2f664c2b9